.github/workflows/ci.yaml

```diff
@@ -19,10 +20,8 @@ jobs:
             install_deps: sudo apt-get install llvm-9-tools llvm-9-dev pkg-config
             path_extension: /usr/lib/llvm-9/bin
 
-    runs-on: ${{ matrix.os }}
-
     steps:
-    - name: Checkout the repository
+    - name: Check out the repository
```

Comment:
isn't `Checkout` here a referring to `git checkout`? and thus the spelling is correct?
Not that it is a big deal either rway

Reply:
Thanks. I don't feel strongly, consistently changed back to `Checkout` everywhere.